rules/windows/credential_access_bruteforce_admin_account.toml

```
@@ -20,27 +20,45 @@ license = "Elastic License v2"
+- If this activity is suspicious, contact the account owner and confirm whether they are aware of it.
+- Examine the source host for derived artifacts that indicates compromise:
+  - Observe and collect information about the following activities in the alert source host:
+    - Attempts to contact external domains and addresses.
+      - Examine the DNS cache for suspicious or anomalous entries.
+        - !{osquery{"query":"SELECT * FROM dns_cache", "label":"Osquery - Retrieve DNS Cache"}}
+    - Examine the host services for suspicious or anomalous entries.
+      - !{osquery{"query":"SELECT description, display_name, name, path, pid, service_type, start_type, status, user_account FROM services","label":"Osquery - Retrieve All Services"}}
+      - !{osquery{"query":"SELECT description, display_name, name, path, pid, service_type, start_type, status, user_account FROM services WHERE NOT (user_account LIKE '%LocalSystem' OR user_account LIKE '%LocalService' OR user_account LIKE '%NetworkService' OR user_account == null)","label":"Osquery - Retrieve Services Running on User Accounts"}}
+      - !{osquery{"query":"SELECT concat('https://www.virustotal.com/gui/file/', sha1) AS VtLink, name, description, start_type, status, pid, services.path FROM services JOIN authenticode ON services.path = authenticode.path OR services.module_path = authenticode.path JOIN hash ON services.path = hash.path WHERE authenticode.result != 'trusted'","label":"Osquery - Retrieve Service Unsigned Executables with Virustotal Link"}}
+- Investigate potentially compromised accounts. Analysts can do this by searching for login events (for example, 4624) to the host which is the source of this activity.
 
 ### False positive analysis
 
 - Authentication misconfiguration or obsolete credentials.
 - Service account password expired.
-- Trust relationship between the primary domain and the trusted domain issue.
-- Infrastructure or availability issue.
+- Domain trust relationship issues.
+- Infrastructure or availability issues.
 
 ### Response and remediation
 
 - Initiate the incident response process based on the outcome of the triage.
-- If the host is a domain controller (DC):
-  - Activate your incident response plan for total Active Directory compromise.
-  - Review the privileges assigned to users that can access the DCs, to ensure that the least privilege principle is being followed and to reduce the attack surface.
-- Isolate the involved hosts to prevent further post-compromise behavior.
+- Isolate the source host to prevent further post-compromise behavior.
+- If the asset is exposed to the internet with RDP or other remote services available, take the necessary measures to restrict access to the asset by not exposing it preferentially. If not possible, limit the access via the firewall to only the needed IP addresses. Also, ensure the system uses robust authentication mechanisms and is patched regularly.
```

I found "by not exposing it preferentially" hard to follow. Does that mean to just restrict/block access to the asset? Maybe the extra phrase is unnecessary then:
```suggestion
- If the asset is exposed to the internet with RDP or other remote services available, take the necessary measures to restrict access to the asset. If not possible, limit the access via the firewall to only the needed IP addresses. Also, ensure the system uses robust authentication mechanisms and is patched regularly.
```
But if the phrasing does add extra intended meaning, consider rephrasing to make a bit clearer.